framework/set/resources/proxy/createMainTF.go

```diff
@@ -0,0 +1,86 @@
+package proxy
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/rancher/tfp-automation/framework/set/resources/proxy/rancher"
+	"github.com/rancher/tfp-automation/framework/set/resources/proxy/rke2"
+	"github.com/rancher/tfp-automation/framework/set/resources/proxy/squid"
+	"github.com/rancher/tfp-automation/framework/set/resources/sanity"
+	"github.com/rancher/tfp-automation/framework/set/resources/sanity/aws"
+)
+
+const (
+	rke2Bastion     = "rke2_bastion"
+	rke2ServerOne   = "rke2_server1"
+	rke2ServerTwo   = "rke2_server2"
+	rke2ServerThree = "rke2_server3"
+
+	rke2ServerOnePublicDNS   = "rke2_server1_public_dns"
+	rke2BastionPublicDNS     = "rke2_bastion_public_dns"
+	rke2ServerOnePrivateIP   = "rke2_server1_private_ip"
+	rke2ServerTwoPublicDNS   = "rke2_server2_public_dns"
+	rke2ServerThreePublicDNS = "rke2_server3_public_dns"
+
+	terraformConst = "terraform"
+)
+
+// CreateMainTF is a helper function that will create the main.tf file for creating an Airgapped-Rancher server.
```

Comment:
`for creating an Airgapped-Rancher server.` => `for creating a Rancher server behind a proxy.`

Reply:
Good call out, resolved.